action/protocol/staking/validations.go

```diff
@@ -39,13 +39,10 @@ func (p *Protocol) validateCreateStake(ctx context.Context, act *action.CreateSt
 	if act.Amount().Cmp(p.config.MinStakeAmount) == -1 {
```

Comment:
feel more natural to call act.SanityCheck() right after if act == nil, since it is the most basic check